device/accton/x86_64-accton_es9618xx-r0/sonic_platform/chassis.py

```diff
@@ -58,6 +58,9 @@ def __initialize_sfp(self):
         for index in range(0, PORT_END):
             sfp = Sfp(index)
             self._sfp_list.append(sfp)
+            api = sfp.get_xcvr_api()
+            if api is not None:
+                api.set_lpmode(False)
```

Comment:
The function '__initialize_sfp()' is called when the object of 'Chassis' is created. There are many python code that imports 'chassis.py' and create 'Chassis' object to manipulate chassis related objects.

If you set high power mode here, it might result in wrong power mode state when users had issued the command to set it in low power mode and it should be kept as is.

When cmis_mgr is enabled, the cmis_mgr will configure the xcvr's power mode to high during the procedure. So the original workaround in the script of xplt package can be removed when cmis_mgr is enabled.